<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PwdGeneratorDialog</class>
 <widget class="QDialog" name="PwdGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_copyToClipboard">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>18</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Kopiuj do schowka</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/copy-solid.svg</normaloff>:/resources/icons/copy-solid.svg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_regeneratePwd">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>18</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Generuj nowe hasło</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/dice-solid.svg</normaloff>:/resources/icons/dice-solid.svg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_togglePwdVisibility">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>18</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Pokaż hasło</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/eye-solid.svg</normaloff>:/resources/icons/eye-solid.svg</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="icon_pwdIcon">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>14</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/icons/about.svg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_newPwdLabel">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nowe hasło:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_newPwd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>116</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>buttonBox</zorder>
  <zorder>btn_copyToClipboard</zorder>
  <zorder>btn_regeneratePwd</zorder>
  <zorder>btn_togglePwdVisibility</zorder>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PwdGeneratorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PwdGeneratorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
